### ISSUE TITLE
Unable to add new blog to ScribeFire. Unable to add new blog to ScribeFire

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#526]
```
What's the problem?

When I try to add a blog to ScribeFire, I immediately receive the following 
message:

"ScribeFire couldn't get the information it needed about your blog. Helpfully, 
your blog returned this message: ScribeFire could not successfully connect to 
your blog."

Actually, I only receive this message in Google Chrome or Opera, but in 
Firefox, no message ever appears (the throbber on the Finish button just keeps 
spinning until I hit the Cancel link).

It's strange, because as I go through the process of adding the blog, 
everything seems correct each step of the way.  The Blog Type and API are 
automatically detected correctly based on the blog URL, and I know my login and 
password are correct as well.

This doesn't appear to be a problem with the blog (but I couldn't say for sure, 
as this is a self-hosted Wordpress API blog which I don't administer, but have 
posting access on), as I can connect to it using BloGTK or Blogilo, two desktop 
clients for Linux.

It simply appears that for whatever reason ScribeFire has stopped working 
correctly sometime over the last couple weeks, as I had previously had a 
working copy of ScribeFire installed, with my blog already configured.  When I 
re-installed my OS (Ubuntu 11.10 or Xubuntu 11.10), suddenly the extension 
started behaving as described.


What browser are you using?

Google Chrome for Linux (v15 stable or v17 unstable)
Firefox for Linux (version 8)
Opera for Linux (version 11.6)

What version of ScribeFire are you running?

version 4.0
```

-----
Original issue reported on code.google.com by `ericcf...@gmail.com` on 12 Dec 2011 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
this is happening in windows vista too
```

-----
Original comment by `asllear...@gmail.com` on 22 Jul 2012 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 4.2 for Chrome
```

-----
Original comment by `scribefi...@gmail.com` on 29 Aug 2013 at 3:28
* Changed state: **Fixed**
